src/components/widgets/ButtonSimple.vue

```diff
@@ -73,6 +79,7 @@ import {
   CornerLeftDownIcon,
   CornerRightDownIcon,
   DownloadIcon,
+  FileDigit,
```

Comment:
Use `FileDigitIcon` to be consistent with other icon imports
```suggestion
  FileDigitIcon,
```